### ISSUE TITLE
Define Schema

### ISSUE THREAD

**Issue by schanzer** [bootstrapworld/pioneers#1]
Start by thinking about what data we want to record for each pioneer. These aren't necessarily the same we want to promote, but it's easier to lose information than gain it.

Some columns to get started:
- [ ] Name
- [ ] Born 
- [ ] Died
- [ ] Country of Origin
- [ ] Contributions
- [ ] Source (from where this data was gathered)
- [ ] Boolean columns for various populations (Female, LGBTQ, Black, Latinx, Native American / Indigenous, East Asian, Indian, etc)

WHEN COMPLETE: hand off a spreadsheet with columns for each piece of data we want. The spreadsheet should contain 2-3 sample rows for people (Grace Hopper, for example!). Trying out the schema on a few sample datapoints will help identify places where the scheme may be under or over-developed.

SIGNOFF: It's critical to get the schema right, so it'll go out to the larger team before you proceed to #2

**Comment by shriram** [comment]
What is the difference between "Did" and "Contributions"? "Did" is too vague to be understandable, surely that's not a good column name.

I was hoping to see the populations columns enumerated. Schema change is hard and annoying.

**Comment by shriram** [comment]
Why 2-3 sample rows? Isn't that the job of https://github.com/bootstrapworld/pioneers/issues/2?

**Comment by shriram** [comment]
For external contributors, you'll want fields to record their social media handles (Fb, Tw, Ig) so we can tag them when we publish.

**Comment by rrajesh1** [comment]
**First Draft of Schema**

https://docs.google.com/spreadsheets/d/1FHJrFlqTqCQwhYWhsI_OuNwb2tQaPmv0EneWHvj1jWw/edit?usp=sharing

Some comments on choices I made. 
1. I chose to include all sources for a given person within a single cell. 

2. I separated the columns “Higher Education” and “Degrees Earned”, even though the information in these columns overlap at times. In the case of Lynn Conway, I found that she studied at M.I.T for a period of time but did not earn a degree from there, so it made sense to separate these columns in order to include a case like this. 

3. I was unsure whether or not a person’s religion should be included as a population we wanted to track. For the two chosen examples, this information was not something that I was able to find. Additionally, I was unsure how many different religions I would end up needing to include if I made each one a boolean column. 

Please let me know what I can do to improve the schema. Thank you!

**Comment by shriram** [comment]
Interesting question. Religion is also something that can vary and can be easy to misrepresent. (Gender can also vary — e.g., Lynn Conway! — but at least fairly public in the end.) I would leave religion out entirely — seems too controversial.

**Comment by schanzer** [comment]
@rrajesh1 I agree with Shriram about removing religion. Don't forget to add a column for social media handles, so that we can thank people who've contributed!

@shriram what do you think about the column for marital status? I'm tempted to leave it out.

**Comment by rrajesh1** [comment]
Got it, that makes sense. I will make those changes.

**Comment by rrajesh1** [comment]
I removed the column "marital status" and added 3 additional columns for Facebook, Twitter, and Instagram handles for external contributors. Is there anything else that should be added/removed before moving to the next issue?

**Comment by schanzer** [comment]
I feel good about these changes. @shriram , are you ready to sign off?

**Comment by shriram** [comment]
Almost. The only thing I don't like is "time of contribution". We don't actually know everything they've done. For all we know, they're actively working right now (if alive*). For instance, Margaret Hamilton is now legendary for her work on Apollo 11, but she is very proud of her software design work from the 70s, which was mostly ignored; would we not list that as part of her "contribution"? I think it's more like "time of fame" than "time of contribution", but nobody on the list would be there if they weren't either famous or deserving of it, so…

In short, what value does this provide? And doesn't it do more harm than good?

* And then, of course, you have people whose co-authors publish collaborative work *after they've died* (e.g., Paul Erdos)…

**Comment by rrajesh1** [comment]
Yes I see how this can be problematic. My reasoning behind that was I wanted to include the time of the specific contributions listed in the contributions column, and I am thinking that I can probably just collapse the two and include the dates within my paragraph explaining those major contributions. Would that be better?

**Comment by shriram** [comment]
My suggestion: leave out dates. It's a complicated issue. When was the contribution "made"? Is that the same as when they actually started *working* on it, which could have been two decades earlier? Also, we're recognizing people who've done work that basically stands the test of time, which means their contributions are, in some sense, timeless. I have this feeling that dates just make everything more complicated. Plus, if we want people to contribute more suggestions, the more things we ask of them, the more work it becomes and so the fewer people contribute.

**Comment by rrajesh1** [comment]
Yes that makes sense, thank you for the feedback. I just made the change and removed that column. Is there anything else to be done before moving to the next issue?

**Comment by shriram** [comment]
I'm happy with it! 👍

**Comment by schanzer** [comment]
Ditto - great work, @rrajesh1 ! On to the next one.
